babel.config.js

```diff
@@ -0,0 +1,67 @@
+/* eslint global-require: 0 */
+
+const developmentEnvironments = ['development', 'test'];
+
+const developmentPlugins = [require('react-hot-loader/babel')];
+
+const productionPlugins = [
+  require('babel-plugin-dev-expression'),
+
+  // babel-preset-react-optimize
+  require('@babel/plugin-transform-react-constant-elements'),
+  require('@babel/plugin-transform-react-inline-elements'),
+  require('babel-plugin-transform-react-remove-prop-types')
+];
+
+module.exports = api => {
+  // see docs about api at https://babeljs.io/docs/en/config-files#apicache
+
+  const development = api.env(developmentEnvironments);
+
+  return {
+    presets: [
+      [
+        require('@babel/preset-env'),
+        {
+          targets: { electron: '2.0.6' },
+          useBuiltIns: true
```

Comment:
According to the [docs](https://babeljs.io/docs/en/babel-preset-env#usebuiltins) can no-longer be true, must be one `"usage"` | `"entry"` | `"false"`. Seems happy defaulting to false for me.

Reply:
@GWStuartClift you're right. Changed this to `"usage"`